api/db/migrate/20250115212958_add_other_description_priority_importance_to_collaborator.rb

```diff
@@ -0,0 +1,7 @@
+class AddOtherDescriptionPriorityImportanceToCollaborator < ActiveRecord::Migration[6.1]
+  def change
+    add_column :collaborators, :other_description, :string
+    add_column :collaborators, :priority, :integer
+    add_column :collaborators, :importance, :integer
+  end
```

Comment:
These new fields should be indexed, I'd make a composite for the order:

```ruby
change_table :collaborators do |t|
  t.index %i[collaboratable_id collaboratable_type priority role importance], name: "index_collaborators_sort_order"
end
```

This will optimize for cases like the view, so we can do `WHERE collaboratable_id = '1234' AND collaboratable_type = 'Text' ORDER BY priority ASC, role ASC importance ASC`